src/main/java/com/easypost/service/BatchService.java

```diff
@@ -97,9 +92,7 @@ public Batch createAndBuy(final Map<String, Object> params) throws EasyPostExcep
      * @throws EasyPostException when the request fails.
      */
     public Batch label(final String id, final Map<String, Object> params) throws EasyPostException {
-        String url = String.format("%s/label", InternalUtilities.instanceURL(Batch.class, id));
-
-        return Requestor.request(RequestMethod.POST, url, params, Batch.class, client);
+        return Requestor.request(RequestMethod.POST, "batches/" + id + "/label", params, Batch.class, client);
```

Comment:
Cleaner vs. waste of a variable, I'll defer to you.

Reply:
I won't block on it, but I don't think it's actually cleaner since you're cramming string interpolation into a function call instead of only giving it finalized parameters.

Reply:
Resolved